explainshell/config.py

```diff
@@ -8,7 +8,7 @@
 
 MAN2HTML = os.path.join(TOOLSDIR, 'w3mman2html.cgi')
 
-MONGO_URI = 'mongodb://localhost'
+MONGO_URI = os.getenv('MONGO_URI', 'mongodb://localhost')
```

Comment:
Would you like to have one file called config.py and config_docker.py?

Maybe I missunderstood something.